.gitignore

```diff
@@ -56,6 +56,7 @@ src/qt/bitcoin-qt.creator.user
 src/qt/bitcoin-qt.files
 src/qt/bitcoin-qt.includes
 
+tags
```

Comment:
Reason for this?

Reply:
This is the tags indexing file, used for quickly navigating to symbols within the project. Many IDEs use it and it updates as changes happen so it is not static.